text/0000-bdfl.md

```diff
@@ -0,0 +1,106 @@
+- Feature Name: `bdfl`
+- Start Date: 2018-04-01
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC ends the quaint process of RFCs and begins the appointment process,
+by referanda, of a benevolent dictator for life.
+
+# Motivation
+[motivation]: #motivation
+
+Everyone can agree that our decision making process,
+of RFCs and building consensus within teams and the community at large,
+while being a nice idea and all,
+is ineffective, slow, and leads to bad decisions.
+
+Instead, we clearly need a strong leader who can make the
+`Result<RightThing™, E>`s happen and unify all `E`s with `!`.
```

Comment:
Worth noting that this BDFL would need to make their first action be to make `™` a valid part of a Rust identifier in order to make this happen.

Reply:
Yes, sadly, the consensus process has been ineffective in that department!

Not even with `#![feature(non_ascii_idents)]` can we have `™`s in our identifiers. Rust is clearly not production ready since companies can't put ™s in their identifiers.

```
error: unknown start of token: \u{2122}
 --> src/main.rs:3:18
  |
3 | struct RightThing™;
  |                  ^
```